tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py

```diff
@@ -47,7 +47,7 @@
             endIndex=7,
             required=True,
             validators=[
-                validators.dateYearIsLargerThan(1998),
+                validators.dateYearIsLargerThan(2019),
```

Comment:
shouldn't we use current year as reference and go a couple of years back? I don't think anyone will add files from 3, 4 year ago and schema_defs are only used at parsing